specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/machineLearningServices.json

```diff
@@ -1228,6 +1228,284 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/Microsoft.MachineLearningServices/workspaces/{workspaceName}/linkedWorkspaces": {
```

Comment:
Do you need pagination? How many entities can exist?

Reply:
there will be less than 10 for typical case, may reach 50 for some extreme cases. As I mentioned to you in the review meeting, we plan to do it w/o pagination in private preview, and will add pagination before public preview.

Reply:
Yes, that's fine